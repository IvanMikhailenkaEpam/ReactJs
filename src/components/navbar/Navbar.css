nav {
    background: url("https://assets.nflxext.com/ffe/siteui/vlv3/783e3b0f-b68e-440e-a980-0e1c19cbe76c/e762d259-b33a-4e43-bcd6-e961c2072ec2/BY-en-20190318-popsignuptwoweeks-perspective_alpha_website_small.jpg");
}
.search-aria {
    width: 100%!important;
    background-color: black!important;
    border: 0;
    border-bottom: 2px solid #F85266;
}

.button-pink {
    background-color: #F85266;
    width: 100px;
    font-weight: bold;
    cursor: pointer;
}

.button-black {
    background-color: #3C3B39;
    width: 100px;
    font-weight: bold;
    cursor: pointer;
}

.search-button {
    left: -15px;
}

.search-button > button {
    background-color: #F85266;
    width: 150px;
    font-weight: bold;
    cursor: pointer;
}


