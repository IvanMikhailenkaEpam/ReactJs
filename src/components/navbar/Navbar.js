import React, { Component } from 'react';
import './Navbar.css';

class Navbar extends Component {
  render() {
    return (
      <nav className="navbar navbar-expand-md navbar-dark bg-dark header">
        <div className="container">
          <div className="row mt-1 w-100">
            <div className="col">
              <span className="pink font-weight-bold">netflixroulette</span>
            </div>
            <div className="col-12 mt-5 white font-weight-bold"><span>FIND YOUR MOVIE</span></div>
            <div className="col-12">
              <form className="col-12 form-inline my-2 my-lg-0 pl-0">
                <input
                  className="form-control mr-sm-2 white search-aria"
                  type="text"
                  placeholder="Search"
                  aria-label="Search"
                />
              </form>
            </div>

            <div className="col-4 mt-3 mb-4 ">
              <span className="mr-3 white font-weight-bold">SEARCH BY</span>
              <button
                className="btn my-2 my-sm-0 white button-pink"
                type="button"
              >
                TITLE
              </button>
              <button
                className="btn my-2 ml-2  my-sm-0 white button-black"
                type="button"
              >
                GENRE
              </button>
            </div>
            <div className="col-2 mt-3 mb-4 ml-auto search-button">
              <button
                className="btn my-2 my-sm-0 white"
                type="button"
              >
                SEARCH
              </button>
            </div>
          </div>
        </div>
      </nav>
    );
  }
}

export default Navbar;
