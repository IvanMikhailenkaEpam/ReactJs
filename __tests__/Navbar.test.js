import React from 'react';
import { shallow } from 'enzyme';
import Navbar from "../src/components/navbar/Navbar";

describe('Navbar', () => {
  it('renders Navbar correctly', () => {
    const component = shallow(<Navbar />);

    expect(component).toMatchSnapshot();
  });
});
