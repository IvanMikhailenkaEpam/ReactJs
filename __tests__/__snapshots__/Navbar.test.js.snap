// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders Navbar correctly 1`] = `
<nav
  className="navbar navbar-expand-md navbar-dark bg-dark header"
>
  <div
    className="container"
  />
</nav>
`;
